tracks/r/exercises/raindrops.md

```diff
@@ -0,0 +1,60 @@
+### Possible solution
+
+This solution is very intuitive and follows step-by-step the instructions in the exercise
+
+```r
+raindrops <- function(number) {
+  rain <- ""
+
+  if (number %% 3 == 0)
+    rain <- append(rain, "Pling")
+  
+  if (number %% 5 == 0)
+    rain <- append(rain, "Plang")
+  
+  if (number %% 7 == 0)
+    rain <- append(rain, "Plong")
+  
+  if (!(number %% 3 == 0 | number %% 5 == 0 | number %% 7 == 0)) # alternatively: if(rain == "")
+    return(as.character(number))
+  
+  paste(rain, collapse = "")
+}
+
+```
+
+The learning goal here is (I guess) to avoid unnecessary conversion of inputs as strings and to account for inputs that are factors of more than one if-statement (e.g. 15).
+
+### Common suggestions
+
+- avoid a string transformation of the input at the beginning
+- go through the task iteratively to account for all possible factors
+- for-loops impact the readability here a bit, I think, so one could point learners to alternatives 
```

Comment:
Do you mean the code examples here with "alternatives"?

Reply:
yes, so either the ones I provided below or if someone has an alternative idea based on the submitted solution (I personally think that building upon the submitted solutions and point people to more efficient ways of solving an exercises is more helpful than "imposing" a specific solution one considers most appropriate)

Reply:
Yeah, good idea! I have the feeling that earlier mentor notes tend towards "imposing", because they contain only 1 solution. Maybe it's fine for the beginner exercises, though?

In case the code examples were copied, how about we link to their sources? Otherwise, I'm hoping for https://github.com/exercism/exercism/issues/4067 to be able to link to an overview page ;-)

Reply:
In case we decide against linking to community solutions, https://github.com/exercism/website-copy/pull/928#discussion_r277176645 would be resolved as well.